mbed_app_esp8266.json

```diff
@@ -16,14 +16,6 @@
         },
         "UBLOX_EVK_ODIN_W2": {
             "target.device_has": ["EMAC"]
-        },
-        "NUCLEO_L476RG": {
-            "wifi-tx": "D8",
-            "wifi-rx": "D2"
-        },
-        "NUCLEO_F401RE": {
-            "wifi-tx": "D8",
-            "wifi-rx": "D2"
```

Comment:
F401RE is still mentioned, so maybe that should stay.

Reply:
F401RE is only mentioned as compatible with IDW wifi boards.
I couldn't get F401RE to run with ESP8266, but the same applies to F429ZI. Do we have some reliable Jenkins configs, where I could look into the configuration, to compare against mine?

Reply:
https://jenkins-internal.mbed.com/view/IP%20Core/job/ipcore-cliapp-test-build/compiler=GCC_ARM,protocol=IPv4,target=NUCLEO_F429ZI,variant=ESP8266/

Reply:
https://github.com/ARMmbed/mbed-os-cliapp/blob/master/mbed_app.wlan0.json

Seems there is no pin changes

Reply:
Actually, these could be removed because they are the default ones: https://github.com/ARMmbed/mbed-os/blob/master/components/wifi/esp8266-driver/mbed_lib.json#L42..L48